### PR TITLE
docs: improve description of enable_jekyll=disable_nojekyll (see #130)

### PR DESCRIPTION
**Pull request by Cito**
This assumes `enable_jekyll` is implemented as (the preferred) alias for `disable_nojekyll` (see #130).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -231,19 +231,19 @@ For more details about `CNAME`, read the official documentation: [Managing a cus
     cname: github.com
 ```
 
-### ⭐️ Disable `.nojekyll`
+### ⭐️ Enable Jekyll
 
-By default, this action adds the `.nojekyll` file to only the `master` and `gh-pages` branches. When the file already exists, this action does nothing.
+If you are using the static site generator Jekyll, set `enable_jekyll` to true.
```
Comment (peaceiris) [inline README.md]:
```suggestion
If you are using the GitHub Pages built-in static site generator Jekyll, set `enable_jekyll` to true.
```

Users who use Jekyll on Docker do not need to set `enable_jekyll` to true.

Comment (peaceiris) [reply README.md]: Sorry, do not care. Jekyll seems not to read files which name starts with an underscore no matter it is built-in, or not.

**Comment by peaceiris** [comment]
Can this description confuse users who use Jekyll on Docker? Actually, they do not need `enable_jekyll: true`. Their pages process may be slow without `.nojekyll`.

**Comment by peaceiris** [comment]
The current description mentions `.nojekyll` file, not the GitHub Pages built-in Jekyll.

I came to think that we do not have to refer to the role of the `.nojekyll` file...

**Comment by peaceiris** [comment]
How about `disable_adding_nojekyll_file` instead of `enable_jekyll`?

File: README.md (hunk 1 above)
Comment (Cito) [reply README.md]: Tried to make it clearer now in the description.

**Comment by Cito** [comment]
> How about `disable_adding_nojekyll_file` instead of `enable_jekyll`?

That's clearer than `disable_nokeyll`, but longer and clumsy, and I still I don't like the double negation that forces you to "think around the corner" as we say in German.

When deploying, users only care about whether Jekyll processing is enabled or not, Thy don't actually care what semaphore mechanism is used to achieve this (the `.nojekyill` file). That's why I mentioned it only in the second paragraph for those to understand the exact details.

**Comment by peaceiris** [comment]
Thanks to your great work, the description became more clear! LGTM.

After merging the implementation of `enable_jekyll` by me, this pull request will be merged into the master. I will release it as v3.5.0 and update the v3 tag.

**Pull request merged**